test-utils/test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/random/IntegerGeneratorTest.java

```diff
@@ -0,0 +1,218 @@
+package uk.gov.justice.services.test.utils.core.random;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class IntegerGeneratorTest {
```

Comment:
if you are generating random numbers, range checking ensures that it works in those ranges and not just in a blanket range.